test/k3s/bootstrap.sh

```diff
@@ -6,6 +6,8 @@ cp -f /var/lib/rancher/k3s/agent/etc/containerd/config.toml /var/lib/rancher/k3s
 cat <<EOF >> /var/lib/rancher/k3s/agent/etc/containerd/config.toml.tmpl
 [plugins."io.containerd.cri.v1.runtime".containerd.runtimes.wasm]
   runtime_type = "$PWD/dist/bin/containerd-shim-$1-v1"
+[plugins."io.containerd.cri.v1.runtime".containerd.runtimes.wasm.options]
```

Comment:
question: why is k3s using `runtmie` while k8s uses `cri`?

Reply:
The latest k3s uses containerd 2.0
https://github.com/containerd/runwasi/pull/891
https://github.com/containerd/containerd/blob/main/docs/cri/config.md#basic-configuration

Reply:
Aha, that's the part I was missing

Reply:
Thanks!